GeoIP2/DatabaseReader.cs

```diff
@@ -147,6 +195,18 @@ public AnonymousIPResponse AnonymousIP(string ipAddress)
         }
 
         /// <summary>
+        ///     Tries to lookup an <see cref="AnonymousIPResponse" /> for the specified IP address.
+        /// </summary>
+        /// <param name="ipAddress">The IP address.</param>
+        /// <param name="response">The <see cref="AnonymousIPResponse" />.</param>
+        /// <returns>A <see cref="bool" /> describing whether the IP address was found.</returns>
+        public bool TryAnonymousIP(string ipAddress, out AnonymousIPResponse response)
```

Comment:
For some of these, there is only a method that takes a `string` IP and for others the method takes only an `IPAddress`. We should have both or at least be consistent.